packages/web/src/app/doc-pages/components/table-doc/table-checkbox-mobile-ceg/table-checkbox-mobile-ceg.component.html

```diff
@@ -31,11 +31,11 @@
   <tbody>
     <tr class="e-table-mobile__checkbox">
       <th class="e-table-mobile__cell e-text-mono" scope="col">
-        <label class="e-checkbox" for="checkbox-second-row-mb">
-          <input type="checkbox" id="checkbox-second-row-mb" />
+        <label class="e-checkbox" aria-label="Velg sak 203413223">
+          <input type="checkbox" />
           <span class="e-checkbox__mark"></span>
         </label>
-        203413222
+        203413223
```

Comment:
Is this number some easter egg?

Reply:
Hahah, no I changed it just so it would be different from the ID on the row above. Just to make it a bit more realistic.